easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2-intel-2020a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'SCons'
+
+name = 'libobjcryst'
+version = '2017.2.2'
+
+homepage = 'https://github.com/diffpy/libobjcryst'
+description = "ObjCryst++ is Object-Oriented Crystallographic Library for C++"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+source_urls = ['https://github.com/diffpy/libobjcryst/releases/download/v%(version)s/']
+sources = ['libobjcryst-%(version)s.tar.gz']
+patches = [('./libobjcryst_custom_sconscript_local.patch', 1)]
```

Comment:
@zarybnicky We usually include the software version for which the patch was created in the patch filename (`ibobjcryst-2017.2.2_...patch`), but maybe that's not needed here since it seems like the patch is just adding a file?